server/services/communityApiService.ts

```diff
@@ -78,4 +79,28 @@ export default class CommunityApiService {
       throw createError(err.status, err, { userMessage: 'Could retrieve staff details from nDelius.' })
     }
   }
+
+  async getResponsibleOfficersForServiceUser(crn: string): Promise<DeliusOffenderManager[]> {
+    const token = await this.hmppsAuthService.getApiClientToken()
+
+    logger.info({ crn }, 'getting offender managers for service user')
+    try {
+      const deliusOffenderManagers = (await this.restClient.get({
+        path: `/secure/offenders/crn/${crn}/allOffenderManagers`,
+        token,
+      })) as DeliusOffenderManager[]
+
+      const responsibleOfficer = deliusOffenderManagers.filter(offenderManager => offenderManager.isResponsibleOfficer)
+
+      if (!responsibleOfficer) {
```

Comment:
is it the concern of this method to throw an error here?

should we not just return the empty list and let the caller handle it?

Reply:
Yeah, I think that makes sense.